0_7_pandas_datetimes.Rmd

```diff
@@ -0,0 +1,329 @@
+---
+jupyter:
+  jupytext:
+    text_representation:
+      extension: .Rmd
+      format_name: rmarkdown
+      format_version: '1.2'
+      jupytext_version: 1.17.2
+  kernelspec:
+    display_name: Python 3 (ipykernel)
+    language: python
+    name: python3
+---
+
+# Pandas Datetimes
+
+We will frequently find ourselves working with data about *time*. Having a specialised representation of time is useful in numerous respects, primarily because it allows us to do "maths with times". For instance, we may want to use subtraction to calculate the distance between two timepoints; we cannot do this if we represent dates/times as string data. As such, Python has a useful module for handling dates and times, creatively named ["Datetime"](https://docs.python.org/3/library/datetime.html).
+
+Datetimes have a (deserved) reputation for being fiddly (we would wager anyone who has encountered them will agree...). Additionally, in contrast to the close relation between Numpy and Pandas shown on the other pages, the way Pandas handles Datetimes is somewhat different to how they are handled in Numpy. The handling of Datetimes in Pandas is more similar to the Datetime library than to Numpy (and Numpy itself differs substantially from the Datetime library). We would argue that Numpy's handling of Datetimes is the fiddliest. 
+
+As such this page will focus on using Datetimes in Pandas (this is also probably the most likely context in which readers will use Datetimes...). 
+
+This page is a *brief* introduction to Datetimes in Pandas - there is much more than be done with Pandas datetimes than we will show here - but these are the essentials.
+
+
+## Datetimes in Pandas Series
+
+First, let's look at how Pandas handles dates. To do this, we'll create a Pandas Series containing some strings representing dates:
+
+```{python}
+import pandas as pd
+
+# create a Series with some strings representing dates
```

Comment:
I notice that you invariably use lower case for the first letter in comments - is this a policy, or just a habit?

Reply:
More habit than policy, but I also think it looks better 😬 - I can alter them (other pages also) if preferred?